src/index.html

```diff
@@ -11,7 +11,164 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+     action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post"
+    >
+      <fieldset class="form-block">
+        <legend>Personal information</legend>
+        <label class="input-block">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            required
+          >
+        </label>
+        <label  class="input-block">
+          Name:
+          <input
+            type="text"
+            name="name"
+            id="name"
+            required
+          >
+        </label>
+        <label  class="input-block">
+          How old are You?
+          <input
+            type="number"
+            id="old"
+            min="1"
+            max="100"
+            value="12"
+            required
+          >
+        </label>
+        <label class="input-block">
+          Full date of birth:
+          <input
+            type="date"
+            name="date"
+          >
+        </label>
+        <label class="input-block"
+          required>
+            I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agreement"
+            id="agree"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form-block">
+        <legend>Registration</legend>
+        <label  class="input-block">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label
+        required>
+          Password:
+          <input
+            type="password"
+            name="password"
+            id="password"
+            required
+            minlength="5"
+            maxlength="20"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form-block">
+        <legend>An interesting fact about you!</legend>
+        <label class="input-block">
```

Comment:
In this case please add separate labels for 'yes' radio and 'no' radio so that by clicking on the corresponding label(Yes or No), the corresponding radio is activated